inc/entity.class.php

```diff
@@ -77,26 +77,28 @@ public function post_purgeItem() {
     * @param $ID           integer     entities ID
     * @param $start        integer     first line to retrieve (default 0)
     * @param $limit        integer     max number of line to retrieve (0 for all) (default 0)
-    * @param $sqlfilter    string      to add a SQL filter (default '')
+    * @param $sqlfilter    array       to add a SQL filter (default [])
     * @return array of vouchers
-   **/
-   static function getAllForEntity($ID, $start = 0, $limit = 0, $sqlfilter = '') {
+    */
+   static function getAllForEntity($ID, $start = 0, $limit = 0, $sqlfilter = []): array {
```

Comment:
`$start` and `$limit` are set to 0 in the only existing call to this method. These parameters could be dropped and related code could be cleaned.